fleetshard/config/config.go

```diff
@@ -28,7 +28,7 @@ type Config struct {
 	CreateAuthProvider   bool          `env:"CREATE_AUTH_PROVIDER" envDefault:"false"`
 	MetricsAddress       string        `env:"FLEETSHARD_METRICS_ADDRESS" envDefault:":8080"`
 	EgressProxyImage     string        `env:"EGRESS_PROXY_IMAGE"`
-	BaseCrdURL           string        `env:"BASE_CRD_URL" envDefault:"https://raw.githubusercontent.com/stackrox/stackrox/%s/operator/bundle/manifests/"`
+	DefaultBaseCRDURL    string        `env:"DEFAULT_BASE_CRD_URL" envDefault:"https://raw.githubusercontent.com/stackrox/stackrox/%s/operator/bundle/manifests/"`
```

Comment:
Consider something like:
```suggestion
	CentralCRDURLTemplate    string        `env:"CENTRAL_CRD_URL_TEMPLATE" envDefault:"https://raw.githubusercontent.com/stackrox/stackrox/{{ .GitRef }} /operator/bundle/manifests/platform.stackrox.io_centrals.yaml"`
		SecuredClusterCRDURLTemplate    string        `env:"SECURED_CLUSTER_CRD_URL_TEMPLATE" envDefault:"https://raw.githubusercontent.com/stackrox/stackrox/{{ .GitRef }} /operator/bundle/manifests/platform.stackrox.io_securedclusters.yaml"`
```

Reply:
Removed the variable and replaced it with a constant. We can overwrite the default via GitOps, there is no need for a config as an env variable, it would be duplicated.